deploy/ui/form.json

```
@@ -107,26 +107,63 @@
           ]
         },
         {
-          "label": "Admin user",
-          "type": "fieldGroup",
+          "label": "Admin user configuration",
+          "type": "dropDown",
           "required": false,
           "jsonPath": "$.spec.commonConfig",
+          "options": [
+            {
+              "value": "secret",
+              "label": "Secret configuration"
+            },
+            {
+              "value": "default",
+              "label": "Username and Password configuration"
+            }
+          ],
           "visible": true,
           "fields": [
             {
-              "label": "Username",
-              "default": "adminUser",
-              "required": false,
-              "description": "Administrative username for RHPAM components",
-              "jsonPath": "$.spec.commonConfig.adminUser",
-              "type": "text"
+              "label": "Secret to use",
+              "type": "fieldGroup",
+              "required": true,
+              "jsonPath": "$.spec",
```

Is in group needed json path, and if so there should be used the commonConfig object as well, right?
```suggestion
              "jsonPath": "$.spec.commonConfig",
```

---

yep